extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -190,6 +195,31 @@ export class AssessmentDetailsHeader {
 
 	// function that create target selection dropdown
 	private createTargetTypeContainer(): azdata.FlexContainer {
+		let targetTypes = [constants.SUMMARY_SQLDB_TYPE, constants.SUMMARY_MI_TYPE, constants.SUMMARY_VM_TYPE];
+		if (this._readonly) {
+			const targetPlatforms = new Set<string>();
+			const issues = this.migrationStateModel._assessmentResults?.issues;
+			for (let j = 0; j < issues.length; ++j) {
+				targetPlatforms.add(issues[j].appliesToMigrationTargetPlatform);
+			}
+
+			for (let i = 0; i < this.migrationStateModel._assessmentResults?.databaseAssessments.length; ++i) {
+				const issues = this.migrationStateModel._assessmentResults?.databaseAssessments[i].issues;
+				for (let j = 0; j < issues.length; ++j) {
```

Comment:
same here, should j be incremented as   j++

Reply:
changed to forEach